### DISCUSSION TITLE
🔴 Poe 输入框无法固定在底部

### DISCUSSION THREAD

**Discussion by Thomas-Harrell** [tw93/Pake#470] [Q&A]
输出文字后，底部的输入框会逐渐向上移动

**Comment by tw93** [comment]
我试了试是这样的，你的不是Mac版本吗？还是Windows版本？
<img width="1213" alt="image" src="https://github.com/tw93/Pake/assets/8736212/bd7501d1-6481-4137-9e3d-b3c08afd39f2">

**Reply by Thomas-Harrell** [reply]
- 你好，我用的是 Mac 版本，Poe 的版本号是 2.0.6

- 下载的链接这里的：[https://github.com/tw93/Pake/releases/download/V2.0.6/Poe.dmg](https://github.com/tw93/Pake/releases/download/V2.0.6/Poe.dmg)

- 我用 pake-cli 自己打包的 Poe 网页也会出现这种状况

<img width="1238" alt="SCR-20230619-thed" src="https://github.com/tw93/Pake/assets/106761803/3acb535b-2fdd-45bc-9c4d-f0d2f6a69955">
<img width="1238" alt="SCR-20230619-thgd" src="https://github.com/tw93/Pake/assets/106761803/36f08847-2de4-4d33-a531-067d190c342d">
<img width="1238" alt="SCR-20230619-thiw" src="https://github.com/tw93/Pake/assets/106761803/f536c755-b356-480b-b5dd-6e1e15464cb8">
<img width="1238" alt="SCR-20230619-thnp" src="https://github.com/tw93/Pake/assets/106761803/8b282491-b50f-466f-be85-741b3d68f322">
